docs/reference/migration/migrate_7_16.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[migrating-7.16]]
+== Migrating to 7.16
+++++
+<titleabbrev>7.16</titleabbrev>
+++++
+
+This section discusses the changes that you need to be aware of when migrating
+your application to {es} 7.16.
+
+See also <<release-highlights>> and <<es-release-notes>>.
+
+* <<breaking_716_monitoring_changes>>
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+[discrete]
+[[deprecated-7.16]]
+=== Deprecations
+
+The following functionality has been deprecated in {es} 7.16 and will be removed
+in 8.0. While this won't have an immediate impact on your applications, we
+strongly encourage you take the described steps to update your code after
+upgrading to 7.16.
+
+NOTE: Significant changes in behavior are deprecated in a minor release and the
+old behavior is supported until the next major release. To find out if you are
+using any deprecated functionality, enable <<deprecation-logging, deprecation
+logging>>.
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking_716_monitoring_changes]]
+==== Monitoring deprecations
+
+[[monitoring-use-ingest-setting-deprecation]]
+.The `use_ingest` setting on Monitoring exporter configurations is deprecated.
+[%collapsible]
+====
+*Details* +
+The `xpack.monitoring.exporters.*.use_ingest` property has been deprecated in 7.16.0 and
+will be removed in 8.0.0. This parameter controls the creation of pipelines for monitoring
+indices that currently have no function.
```

Comment:
The parameter currently has no function? Or it creates pipelines that have no function?